build/config.props

```diff
@@ -9,10 +9,12 @@
     <SemanticVersion Condition=" '$(SemanticVersion)' == '' ">$(MajorNuGetVersion).$(MinorNuGetVersion).$(PatchNuGetVersion)</SemanticVersion>
     <VsTargetBranch>lab/d16.$(MinorNuGetVersion)stg</VsTargetBranch>
     <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d16.$(MinorNuGetVersion)</VsTargetBranch>
-    <CliTargetBranch Condition="'$(OverrideCliTargetBranch)' != ''">$(OverrideCliTargetBranch)</CliTargetBranch>
-    <CliTargetBranch Condition="'$(OverrideCliTargetBranch)' == ''">master</CliTargetBranch>
-    <SdkTargetBranch Condition="'$(OverrideCliTargetBranch)' != ''">$(OverrideCliTargetBranch)</SdkTargetBranch>
-    <SdkTargetBranch Condition="'$(OverrideCliTargetBranch)' == ''">master</SdkTargetBranch>
+    <CliTargetBranch1 Condition="'$(OverrideCliTargetBranch)' != ''">$(OverrideCliTargetBranch)</CliTargetBranch1>
+    <CliTargetBranch1 Condition="'$(OverrideCliTargetBranch)' == ''">release/2.2.2xx</CliTargetBranch1>
+    <CliTargetBranch2>release/2.1.6xx</CliTargetBranch2>
+    <SdkTargetBranch1 Condition="'$(OverrideCliTargetBranch)' != ''">$(OverrideCliTargetBranch)</SdkTargetBranch1>
+    <SdkTargetBranch1 Condition="'$(OverrideCliTargetBranch)' == ''">release/2.2.2xx</SdkTargetBranch1>
+    <SdkTargetBranch2>release/2.1.6xx</SdkTargetBranch2>
```

Comment:
we should add a way to override both branches.

Reply:
Current override was added to allow running new tests against latest dotnet build to verify, which will continue to work even with this change. I don't see any requirement to run against multiple new channels. So I dont think we need to override both branches or will we never need to do that.. but in case, we ever need it, we'll update then.

Reply:
I know that, but the initial motivation to do that was to test and then ultimately also creating PRs against that branch. Current changes will keep the same behavior but I don't see a point where we need to override both the branches.

But as you mentioned this isn't a big deal, i'll add another argument to override both